### DISCUSSION TITLE
Cross compile failed from 'Macos:armv8' to 'Android:armv8'

### DISCUSSION THREAD

**Discussion by 97Simei** [duckdb/duckdb#6352] [Q&A]
Hi, I am recently working on porting duckdb into the Android platform. 
I met some problems as below:

When I am compiling the duckdb(> v0.6.1), I met the problem with "undeclared identifier 'pthread_getaffinity_np'"  as below:
![Screenshot 2023-02-17 at 12 08 30](https://user-images.githubusercontent.com/86485345/219633388-2e9efbb3-e118-495d-9f4f-f2527c27f8f3.png)

This problem happens both on the Android Studio NDK build process and Conan cross-build process. 

So far, duckdb v0.5.1 works fine on Conan cross-build but still failed on Android Studio.

**Comment by papparapa** [comment]
Disabling jemalloc may solve the problem.
e.g. expanding `make release` in Makefile and modifying it

```shell
$ mkdir -p build/release && cd build/release && cmake -DBUILD_JEMALLOC_EXTENSION=FALSE -DBUILD_PARQUET_EXTENSION=TRUE -DCMAKE_BUILD_TYPE=Release ../.. && cmake --build . --config Release
```
